companies/mapillary-com.json

```diff
@@ -0,0 +1,33 @@
+{
+    "relevant-countries": [
+        "all"
+    ],
+    "name": "Mapillary AB",
+    "runs": [
+        "Mapillary Inc.",
+        "Mapillary GmbH"
+    ],
+    "address": "Bredgatan 4\n211 30 Malmö\nSweden",
+    "email": "support@mapillary.com",
+    "web": "https://www.mapillary.com",
+    "sources": [
+        "https://www.mapillary.com/about",
+        "https://www.mapillary.com/privacy"
+    ],
+    "required-elements": [
+        {
+            "desc": "Name",
+            "type": "name"
+        },
+        {
+            "desc": "Address",
+            "type": "address"
+        },
+        {
+            "desc": "Username",
+            "type": "input"
+        }
+    ],
+    "request-language": "en",
+    "suggested-transport-medium": "email"
```

Comment:
With a `support@` email, we can unfortunately not suggest email as the transport medium.

Reply:
[Their website](https://www.mapillary.com/privacy) states 

> You can always contact Mapillary at support@mapillary.com if you have any questions regarding your personal data processed under joint controllership, or if you wish to exercise any of your rights under this Policy or the GDPR. 

Should I remove the suggested transport medium in these cases?

Reply:
I agree that this is a little debatable. But our policy is to only suggest email for addresses that clearly *only* go to the privacy team/data protection officer (or at least legal team). If there is a general support mail that is also read by the DPO, we just don't suggest a TM at all.